extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -838,6 +839,28 @@ export class Bind {
     });
   }
 
+  /*
+   * Form implementations are not filled in until ampdoc is ready.
+   * So we must wait for that to finish before scanning form elements
+   * for dynamic components.
+   * @return {!Promise}
+   * @private
+   */
+  getAmpFormInitializationPromise_() {
+    if (!this.formInitializationPromise_) {
+      const p = ampFormServiceOrNullForDoc(this.ampdoc).then(ampFormService => {
```

Comment:
If this returns null, calling `then()` will throw an undefined property error right?